tss/error.go

```diff
@@ -17,11 +17,16 @@ type Error struct {
 	round    int
 	victim   *PartyID
 	culprits []*PartyID
+
+	trackingId []byte // optional.
```

Comment:
what is this set to? there's already party id + round in the error, is it the message digest? Maybe just call it that way then.

Reply:
We put the message digest there, but conceptually it is a trackingID, so I think it is more fitting, especially since we have called it in other locations as trackingID